Makefile

```diff
@@ -185,6 +202,10 @@ check-fast: lint $(PYPY) $(PY36) $(TOX)
 check-rst: $(RSTLINT)
 	$(RSTLINT) *.rst
 
+check-ophidian: $(TOX)
+	cd ophidian &&  $(TOX) -e ophidian-py27
+	cd ophidian &&  $(TOX) -e ophidian-py36
```

Comment:
Also, minor: `ophidian-py36` should be a variable near the top, so when we bump the `BEST_PY3` version, we bump ophidian as well.